django_filters/filters.py

```diff
@@ -87,6 +87,16 @@ def __init__(self, field_name=None, lookup_expr='exact', *, label=None,
             "expressions. Use the `LookupChoiceFilter` instead. See: " \
             "https://django-filter.readthedocs.io/en/master/guide/migration.html"
 
+    def bind(self, attr, parent):
```

Comment:
info: When used with `django-rest-framework-filters` this function collides https://github.com/philipn/django-rest-framework-filters/blob/604327f76af5d006e443e4e00c6599847216d286/rest_framework_filters/filters.py#L62

Reply:
Thanks for catching that. I'll updated drff, given that `RelatedFilter.bind` was introduced in 1.0 and hasn't been released yet (note that I maintain drff).